sdk-workflows/src/main/java/io/dapr/workflows/client/WorkflowMetadata.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2023 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.workflows.client;
+
+import com.microsoft.durabletask.FailureDetails;
+import com.microsoft.durabletask.OrchestrationMetadata;
+import com.microsoft.durabletask.OrchestrationRuntimeStatus;
+import java.time.Instant;
+
+/**
+ * Represents a snapshot of an workflow instance's current state, including
+ * metadata.
+ */
+public class WorkflowMetadata {
```

Comment:
Dotnet implementation used `WorkflowState` https://github.com/dapr/dotnet-sdk/blob/master/src/Dapr.Workflow/WorkflowState.cs#L66

As a developer, state of my workflow is a better context than metadata. A state means the data from the running process. Metadata would mean something like data from a storage.

Do you think it would be better to use `State` instead of `Metadata`?

Reply:
Yeah I saw it on the dotnet implementation, I tried to keep the same naming as DurableTask.
But I agree with you WorkflowState is clearer., I will change it

Reply:
Done